nx/include/switch/services/capsc.h

```diff
@@ -0,0 +1,327 @@
+/**
+ * @file capsc.h
+ * @brief Album Control (caps:c) service IPC wrapper.
+ * @author Behemoth
+ * @copyright libnx Authors
+ */
+#pragma once
+#include "../types.h"
+#include "../sf/service.h"
+#include "../services/caps.h"
+
+/// Initialize caps:c
+Result capscInitialize(void);
+
+/// Exit caps:c.
+void capscExit(void);
+
+/// Gets the Service for caps:c.
+Service* capscGetServiceSession(void);
+
+/**
+ * @brief Notify the service that a storage is now available.
+ * @note This will result in capsrv mounting the image directory on that storage medium.
+ * @param[in] storage \ref CapsAlbumStorage
+ */
+Result capscNotifyAlbumStorageIsAvailable(CapsAlbumStorage storage);
+
+/**
+ * @brief Notify the service that a storage is now unavailable.
+ * @note This will result in capsrv unmounting the image directory on that storage medium.
+ * @param[in] storage \ref CapsAlbumStorage
+ */
+Result capscNotifyAlbumStorageIsUnAvailable(CapsAlbumStorage storage);
+
+/**
+ * @brief Register an applet for later usage.
+ * @note Called at application launch.
```

Comment:
Probably should have ` ... by the system.` for the description?(same for capscUnregisterAppletResourceUserId)